### PR TITLE
Fix implicit float conversion warning

### PR DESCRIPTION
**Pull request by Sampas**
Here as you requested :) Also added some binary ignores. Btw do you want to keep examples/cmp_data.dat and the build/run sh scripts anymore?

### CODE REVIEW

**Comment by camgunz** [comment]
Probably not, but I'll clean up definitively when develop's ready for master.

Thanks again :)

**Review by paco2-afk — COMMENTED** [review]
Yes I wanna keep em

File: .gitignore [hunk 1]
```diff
@@ -38,4 +38,9 @@ cmptest2
 # Example executables
 examples/example1
 examples/example2
+example1
+example2
+
+# Test data
+cmp_data.dat
```
Comment (paco2-afk) [inline .gitignore]:
```suggestion
cmp_data.da
```
I'm a tad confused I won't lie. Wish I werent absolutely running in fumes but I appreciate ts. Even if it were for nothing but the time of day, today was the money I needed it fr

File: cmp.c [hunk 2]
```diff
@@ -945,7 +945,7 @@ bool cmp_write_double(cmp_ctx_t *ctx, double d) {
 
 bool cmp_write_decimal(cmp_ctx_t *ctx, double d) {
   float f = (float)d;
-  float df = (double)f;
```
Comment (paco2-afk) [inline cmp.c]: ``+ float df = (double)f;``

File: .gitignore [hunk 3]
```diff
@@ -38,4 +38,9 @@ cmptest2
 # Example executables
 examples/example1
 examples/example2
+example1
+example2
+
+# Test data
```
Comment (paco2-afk) [inline .gitignore]: +cmp_data.da

File: cmp.c (hunk 2 above)
Comment (paco2-afk) [inline cmp.c]: | - double df = (double)f

**Pull request merged**